collection-hooks.js

```diff
@@ -125,8 +125,8 @@ CollectionHooks.initOptions = (options, pointcut, method) =>
 CollectionHooks.extendOptions = (source, options, pointcut, method) =>
   ({ ...options, ...source.all.all, ...source[pointcut].all, ...source.all[method], ...source[pointcut][method] })
 
-CollectionHooks.getDocs = function getDocs (collection, selector, options) {
-  const findOptions = { transform: null, reactive: false } // added reactive: false
+CollectionHooks.getDocs = function getDocs (collection, selector, options, fetchFields) {
+  const findOptions = { transform: null, reactive: false, fields: fetchFields || {} } // added reactive: false
```

Comment:
If fields was set we would add _tenantId (because _id is automatically fetched, but we need _tenantId as well), which would make fields not-empty, causing it to take effect.

Hence only setting fields if there is actually fields in there, we resolve this issue.